modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -36,7 +37,6 @@ object GlobalL0Service {
   ](
     l0GlobalSnapshotClient: L0GlobalSnapshotClient[F],
     globalL0ClusterStorage: L0ClusterStorage[F],
-    lastGlobalSnapshotStorage: LastSnapshotStorage[F, GlobalIncrementalSnapshot, GlobalSnapshotInfo],
```

Comment:
Removing it is a step back in my opinion. I think it's better to have `GlobalL0Service` more atomic the way it was. Allowing injecting ordinal from outside may lead to errors. This service always downloads GlobalSnapshots hence `lastGlobalSnapshotStorage` is required.

Reply:
Yeah. The reason why I did it previously is because I need `pullLatestSnapshot` method from `GlobalL0Service` and that method doesn't need any `lastGlobalSnapshotStorage` to be instantiated. So I needed to either:
- extract storage from `GlobalL0Service`
or - instantiate `lastGlobalSnapshotStorage` on L0 for no reason
or - extract `pullLatestSnapshot` from `GlobalL0Service` to reuse it

Of course I broke DAG which is another story.
Anyway, I will revert that and use different approach.